todo-app/src/Tasks.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import info from './info';
+import Done from './Done-action';
+
+
+export default class Tasks extends React.Component {
+    render() {
+        const { text, done } = this.props.task;
+        return (
+            <div>
+                <div>
+                    <info text={text} />
```

Comment:
React guidelines specify that custom components as in this case, should be named with [first capital letter](https://reactjs.org/docs/components-and-props.html#rendering-a-component). So `Info` instead of `info`.